frontend/components/vault/VaultsText.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+    <div class="flex flex-col space-y-8 py-8">
+        <WhatIsMakerProtocol v-if="isExplanationsShown" class="TextBlock"></WhatIsMakerProtocol>
+        <template v-if="isExplanationsShown">
+            <TextBlock title="What are vaults?" class="TextBlock">
+                A vault is a main source of DAI in the MakerDAO protocol: anyone is able to deposit supported
+                collateral into the system to get stablecoin in return and receive fees for the provided liquidity. The
+                place where the collateral is deposited is called the vault and is solely managed by the person who
+                deposited it, unless the deposited collateral amount no longer covers the withdrawn DAI.</TextBlock
+            >
+            <TextBlock title="When are vaults at risk?" class="TextBlock">
+                Vaults become at risk when the amount of DAI withdrawn from the vault is getting too close to the true
+                value of the collateral in the vault. When market conditions change and the value of a deposited token
+                becomes less, users have to either return part of the withdrawn DAI or add collateral. If the original
+                owner of the vault is inactive, part of the vault can be liquidated after a certain threshold by anyone
+                else to cover missing DAI.
```

Comment:
- [ ] Proposal with some re-phrasings: 
" Vaults become at risk when true value of the collateral in the vault is getting too close to the value of DAI withdrawn from the vault and hence the vault is no longer over collateralised enough. When market conditions change and the value of a deposited token decreases, users have to either return part of the withdrawn DAI or add collateral. If the original owner of the vault doesn't act in this situation, the vault can be liquidated by anyone to cover the missing DAI."